lib/fauna/client.rb

```diff
@@ -104,10 +104,23 @@ def delete(path, data = {})
     # for information on constructing queries.
     #
     # +expression+:: A query expression
+    # +expr_block+:: May be provided instead of expression. Block is used to build an expression with Fauna.query.
+    #
+    # Example using expression:
+    #
+    # <code>client.query(Fauna::Query.add(1, 2, Fauna::Query.subtract(3, 2)))</code>
+    #
+    # Example using block:
+    #
+    # <code>client.query { add(1, 2, subtract(3, 2)) }</code>
     #
     # :category: Query Methods
-    def query(expression)
-      post('', expression)
+    def query(expression = nil, &expr_block)
```

Comment:
This is being documented under "Miscellaneous Functions", making it look like it's part of the Query module. Maybe this code should be moved to a different file.